omod/src/main/java/org/openmrs/module/fhir2/providers/LocationFhirResourceProvider.java

```diff
@@ -47,4 +50,14 @@ public Location getLocationById(@IdParam @NotNull IdType id) {
 		}
 		return location;
 	}
+
+	@Search
+	@SuppressWarnings("unused")
+	public Location getLocationByName(@RequiredParam(name = Location.SP_NAME) StringParam name) {
```

Comment:
I understand that the name is unique from the database.
So this qualifies to be a Read operation **`@Read`**